src/views/components/loader.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * View: Loader
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/components/loader.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+if ( empty( $text ) ) {
+	$text = $this->get( 'text' ) ?: 'Loading...';
```

Comment:
```suggestion
	$text = $this->get( 'text' ) ?: __( 'Loading...', 'event-tickets' );
```

Reply:
Also I keep confusing ?: and ?? and it's PHP version compatibility :) had to double check again!

Reply:
Elvis!